src/test/java/com/github/algobot76/surabaya/AnalyzerTests.java

```diff
@@ -0,0 +1,95 @@
+package com.github.algobot76.surabaya;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.github.algobot76.surabaya.util.Analyzer;
+import com.github.algobot76.surabaya.util.Project;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.core.io.Resource;
+
+@SpringBootTest
+public class AnalyzerTests {
+
+	static final String testProjectDir = "src/test/testProjects/";
```

Comment:
I think it could be better if we place those input files in the `resources` directory under `test`.